packages/components/src/components/marketplace-list.tsx

```diff
@@ -158,3 +132,111 @@ export const MarketplaceList = ({
     </section>
   );
 };
+
+export interface MarketplaceListItemProps extends React.ComponentPropsWithoutRef<'a'> {
+  item: IMarketplaceItemProps;
+}
+
+export function MarketplaceListItem({ item, ...rest }: MarketplaceListItemProps) {
+  return (
+    <Anchor
+      {...item.link}
+      {...rest}
+      className={cn(
+        'hive-focus flex flex-row gap-4 rounded-2xl border border-transparent bg-neutral-50 p-6 @container hover:border-neutral-200/50 hover:bg-neutral-100 @lg:gap-6 dark:bg-neutral-800 dark:hover:border-neutral-700 dark:hover:bg-neutral-700/50 [.green_&]:bg-green-900 [.green_&]:hover:border-green-700/50 [.green_&]:hover:bg-green-800/75',
+        item.link.className,
+      )}
+    >
+      <div
+        className={cn(
+          'size-16 shrink-0 rounded-lg bg-[--bg] @lg:size-16 @2xl:size-[92px] [.green_&]:[background:linear-gradient(135deg,_#68A8B6_0%,_#3B736A_100%)]',
+        )}
+      >
+        <Image
+          {...item.image}
+          width="92"
+          height="92"
+          className="aspect-square rounded-lg object-contain"
+        />
+      </div>
+      <div className="flex flex-col">
+        <h3 className="m-0 line-clamp-2 font-medium text-[--fg] @lg:text-2xl">{item.title}</h3>
+        <div className="mb-2 line-clamp-3 text-sm text-[--fg-80] @lg:text-base">
+          {item.description}
+        </div>
+        {item.tags && item.tags.length > 0 && (
```

Comment:
```suggestion
        {item.tags?.length > 0 && (
```

Reply:
```
'item.tags.length' is possibly 'undefined'.ts(18048)
```

`undefined > 0` is `false`, so this is gonna work at runtime, but it doesn't really work with strict typechecking, and adding a non-null assertion violates `@typescript-eslint/no-non-null-asserted-optional-chain`